packages/context-editor/src/components/AttributePanel.tsx

```diff
@@ -284,6 +344,7 @@ export function AttributePanel({ menuHidden }: { menuHidden: boolean }) {
 					transition: position.top === 0 ? "" : `right ${animationTimeMS}ms linear`,
 				}}
 			/>
-		</>
+		</>,
+		container
```

Comment:
I see now that the reference to the container DOM node is necessary here for the portal. Could you help me understand why a portal is used here?

Reply:
Ah! Just saw your comment about the nested form issue. Such a frustrating thing. I guess we don't really need to use proper `<form>` elements anywhere since our submit handlers are generally invoking JS code on submit (i.e. don't use the native `method` or `action` attributes for the forms). But I guess it's a good thing to use form elements semantically?

Anyways, the portal solution seems like a decent one, I'd just prefer that we pass a ref instead of querying for the element by ID.